TimerCalculatorUtility/TimerCalculatorUtility/ViewModels/CalculatorViewModel.cs

```diff
@@ -0,0 +1,76 @@
+﻿
+using System;
+using System.ComponentModel;
+using System.Data;
+using System.Text;
+using Xamarin.Forms;
+
+namespace TimerCalculatorUtility.ViewModels
+{
+    public class CalculatorViewModel : INotifyPropertyChanged
+    {
+        private StringBuilder _readedString = new StringBuilder();
```

Comment:
Rename it to just string builder. It's more understandable when you read code :)

Reply:
Also, if you don't reassign this variable, make it readonly, so nothing can change it.

Reply:
> Also, if you don't reassign this variable, make it readonly, so nothing can change it.

I reassign it in "DeletelastSymbol()" method, so i can't(